content/pda.md

```diff
@@ -0,0 +1,523 @@
+# PDAs
+
+# Lesson Objectives
+
+_By the end of this lesson, you will be able to:_
+
+- Explain Program Derived Addresses (PDAs)
+- Explain various use cases of PDAs
+- Describe how PDAs are derived
+- Use PDA derivations to locate and retrieve data
+
+# TL;DR
+
+- A Program Derived Address (PDA) is derived from a program ID and an optional list of seeds.
+- PDAs are designed to be controlled by the program they are derived from
+- PDAs provide a deterministic way to find data stored on accounts
+- Programs can sign instruction for PDAs derived from its program ID
+
+## What is a Program Derived Address
+
+Program Derived Addresses (PDAs) are effectively account addresses designed to be signed for by a program rather than a secret key. As the name suggests, PDAs are derived using a program ID and an optional set of seeds. A PDA has the same form as a Solana public key, expect they are ensured to not be on the ed25519 curve and thus have no associated private key.
+
+PDAs serve two main functions:
+
+1. Provide a deterministic way to find the address of an account
+2. Authorize a program to sign for a PDA in the same way a user may sign with the private key associated with their public key.
+
+In this lesson we will focus on using PDAs to store data. We will discuss signing with a PDA more thoroughly in a future lesson where we cover Cross Program Invocations (CPIs).
+
+### Finding PDAs
+
+PDAs are not technically created. Rather, they are _found_ or _derived_ based on a program ID and one or more input seeds. Regular Solana keypairs lie on the ed25519 Elliptic Curve and have public/private keys. PDAs are addresses that lie _off_ the ed25519 Elliptic curve and do not have a corresponding private key. The details of elliptic curve cryptography are outside the scope of this lesson, but for now it is sufficient to understand that PDAs are 32 byte strings that look like public keys without an associated private key.
```

Comment:
@ZYJLiu I may be wrong, but isn't it that PDAs don't have a *valid* corresponding private key?  i.e. The y _do_ have a .secretKey, it just isn't on the curve and can't be used for signing stuff, but there is a Uint8Array 64 value in the PDAs keypair.  IIRC, when I looked at the guts of the PDA there was an array w/64 8-bit values in it 🤷🏻‍♂️

Reply:
For the purposes of the PR, this conversation is resolved, but I am leaving it open until I have a moment to verify :)